parquet/tests/variant_integration.rs

```
@@ -144,10 +144,7 @@ variant_test_case!(39);
 variant_test_case!(40, "Unsupported typed_value type: List(");
 variant_test_case!(41, "Unsupported typed_value type: List(");
 // Is an error case (should be failing as the expected error message indicates)
-variant_test_case!(
-    42,
-    "Expected an error 'Invalid variant, conflicting value and typed_value`, but got no error"
```

This was just flat out wrong, swallowing the error message that correctly identifies a problem 🤦